fw/http.h

```diff
@@ -470,6 +473,7 @@ struct tfw_http_req_t {
 	TfwHttpCond		cond;
 	TfwMsgParseIter		pit;
 	TfwStr			userinfo;
+	TfwStr			*uri_host;
```

Comment:
This new member is used in boolean expressions only, so we should just use `flags` from `TFW_HTTP_MSG_COMMON` and save the 8 bytes